custom_components/spotcast/__init__.py

```diff
@@ -178,7 +185,14 @@ def start_casting(call: ha_core.ServiceCall):
         category = call.data.get(CONF_SPOTIFY_CATEGORY)
         country = call.data.get(CONF_SPOTIFY_COUNTRY)
         limit = call.data.get(CONF_SPOTIFY_LIMIT)
-        search = call.data.get(CONF_SPOTIFY_SEARCH)
```

Comment:
note that "search" is now gone completely, it is supplanted by some combination of all the fields below (for example, track + artist, or show + episode, etc)